Packages/vcs/vcs/utils.py

```diff
@@ -1666,13 +1666,25 @@ def creategraphicsmethod(gtype, gname='default', name=None):
     return copy_mthd
 
 
+# Returns the float value for datawc_...
+# datawc_ can be a float or a cdtime.reltime
+# TODO: Investigate why datawc is converted to a cdtime.reltime
+def getDataWcValue(v):
+    if (type(v) is type(cdtime.reltime(0, 'months since 1900'))):
```

Comment:
@danlipsa why `months since 1900`, we should use the `gm.datawc_units` and `gm.datawc_calendar`